src/reactive-elements.js

```diff
@@ -33,9 +33,12 @@ function reactiveElements(elementName, ReactComponent, options) {
       if (useShadowDom) self.attachShadow({ mode: 'open' });
 
       const observer = new MutationObserver(() => {
-        ReactDOM.unmountComponentAtNode(getRenderRoot(self, useShadowDom));
         const props = utils.getProps(self);
-        props.children = utils.getChildren(self);
```

Comment:
can we get rid of `utils.getChildren` now ? :)

Reply:
No, it's used in the _initial_ render (where we _do_ want it to be destructive, as we're about to move the DOM elements from the current DOM node they're in, to React for it to render)

I think very possibly `getChildren` as a name could stand to be improved, though, to make its destructive nature a bit clearer. Open to naming suggestions there, though